FogCreek/Glitch-Community
# Petalite pot: hookify teams page

## Links
* Remix link: https://petalite-pot.glitch.me

## Changes:
* convert TeamPage to function component
* convert TeamEditor to hook
* clean up redundant methods

## How To Test:
* log into https://petalite-pot.glitch.me
* create a team
* interact with the team page (e.g. add a project) -- it should work the same as before
